ui/package.json

```diff
@@ -1,15 +1,20 @@
 {
-  "name": "webapp",
+  "name": "ui",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
```

Comment:
Do we need react testing dependencies?

Reply:
At least for now - we don't. But I thought there's no harm in keeping them if we decide to write some ui tests (unlikely). If you think it's redundant and it's easy to remove - just do it 🙂